tasks.c

```diff
@@ -938,7 +938,9 @@ static void prvYieldForTask( TCB_t * pxTCB,
             {
                 if( xDecrementTopPriority != pdFALSE )
                 {
-                    uxTopReadyPriority--;
+                    if(uxTopReadyPriority > 0) {
+                        uxTopReadyPriority--;    
+                    }
```

Comment:
The formatting should be something like the following (to adhere to the Kernel's formatting style)
```suggestion
                    if( uxTopReadyPriority > 0 )
                    {
                        uxTopReadyPriority--;    
                    }
```